Mozu.Api.ToolKit/Readers/BaseReader.cs

```diff
@@ -16,7 +16,7 @@ public abstract class BaseReader
         public string Q { get; set; }
         public int? QLimit { get; set; }
         public int? TotalCount { get; protected set; }
-
+        public string Mode { get; set; } 
```

Comment:
This field is specific to order reader. Move the field to order reader instead of base.

Reply:
Moved **Mode** parameter from BaseReader to OrderReader